client/src/components/Reviews/ReviewList/ReviewList.jsx

```diff
@@ -2,29 +2,86 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReviewListEntry from './ReviewListEntry/ReviewListEntry.jsx';
 
+const SORT_BY = ['Relevant', 'Helpful', 'Newest'];
+
 class ReviewList extends React.Component {
-  constructor (props) {
+  constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      listLength: 2,
+      sortBy: SORT_BY[0]
+    };
+    this.incrementListLength = this.incrementListLength.bind(this);
+    this.handleSelectorChange = this.handleSelectorChange.bind(this);
+  }
+
+  incrementListLength() {
+    let { listLength } = this.state;
+    listLength += 2;
+    this.setState({ listLength });
+    // TODO: make GET request for next page of reviews
+    // if listLength >= .75 reviews.length
   }
 
-  render () {
-    let { reviews } = this.props;
+  handleSelectorChange(e) {
+    let idx = e.target.value;
+    this.setState({ sortBy: SORT_BY[idx] });
+    // TODO: make GET request with new sort param
+  }
+
+  render() {
+    let { reviews, starFilter } = this.props;
+    let { listLength } = this.state;
     return (
-      <div style={{flexDirection: 'column', margin: '10px'}}>
-        {reviews.map(review => {
-          return (
-            <ReviewListEntry key={review.review_id} review={review}/>
-          );
-        })}
-      </div>
 
+      <div className="review-list-container">
+        <span>
+          {`${reviews.length} reviews, sorted by `}
+          {/* SORT BY DROPDOWN SELECTOR */}
+          <select onChange={this.handleSelectorChange}>
+            {SORT_BY.map((order, idx) => {
+              return (
+                <option
+                  key={`sort-by-${order.toLowerCase()}`}
+                  value={idx}>
+                  {order}
+                </option>
+              );
+            })}
+          </select>
```

Comment:
I suggest breaking this select element into another component to make ReviewList a smaller component.